# XML output to test/ref/exampleXmlVerbose.xml
# Started on 01/30/16 22:19:37
# Starting class: TestTiti
# Starting test: TestTiti.test1_withFailure
#   Failure:  example_with_luaunit.lua:104: expected: 2, actual: 1
# Starting test: TestTiti.test2_withFailure
#   Failure:  example_with_luaunit.lua:113: expected: "bof"
#   actual: "hop"
# Starting test: TestTiti.test3
# Starting class: TestToto
# Starting test: TestToto.test1_withFailure
#   Failure:  example_with_luaunit.lua:21: expected: 2, actual: 1
# Starting test: TestToto.test2_withFailure
#   Failure:  example_with_luaunit.lua:30: expected: "bof"
#   actual: "hop"
# Starting test: TestToto.test3
# Starting test: TestToto.test4
#   Failure:  example_with_luaunit.lua:43: Received the not expected value: 1
# Starting test: TestToto.test5
#   Failure:  example_with_luaunit.lua:49: expected: false or nil, actual: 1
# Starting test: TestToto.test6
#   Failure:  example_with_luaunit.lua:57: expected: false, actual: nil
# Starting test: TestToto.test7
#   Failure:  example_with_luaunit.lua:63: expected: {1, 2, three=3}
#   actual: {1, 2}
# Starting test: TestToto.test8a
#   Error:  example_with_luaunit.lua:81: Bouhouhoum error!
# Starting test: TestToto.test8b
#   Error:  example_with_luaunit.lua:81: Bouhouhoum error!
# Starting test: test1_withFailure
#   Error:  example_with_luaunit.lua:129: assertion failed!
# Starting test: test2_withFailure
#   Error:  example_with_luaunit.lua:135: assertion failed!
# Starting test: test3
# Ran 15 tests in 0.015 seconds, 3 successes, 8 failures, 4 errors
